osler/templates/inventory/inventory-main.html

```diff
@@ -35,25 +35,22 @@ <h2>Drug Inventory</h2>
                 <td><b>{{ drug.category }}</b></td>
                 <td><b>{{ drug.stock }}</b></td>
                 <td>{{ drug.lot_number }}</td>
-                <td>{% if drug.dose %}
-                        {{ drug.dose }}
-                    {% endif %}
-                    {% if drug.unit %}
-                        {{ drug.unit }}
-                    {% endif %}</td>
-                <td>{% if drug.expiration_date %}
-                        {{ drug.expiration_date }}
-                    {% endif %}</td>
-                <td>{% if drug.manufacturer %}
-                        {{ drug.manufacturer }}
-                    {% endif %}
+                <td>{{ drug.dose }}{{ drug.unit }}</td>
+                <td>{% if drug.expired %}
+                  <b style="color:red;">{{ drug.expiration_date }}</b>
```

Comment:
You should use Bootstrap's [pre-built color indicators](https://getbootstrap.com/docs/3.3/css/#helper-classes-colors), so it's consistent across the site. (likewise below with gold for warning).

Reply:
I tried Bootstrap's pre-built colors, but the color didn't come out good. The color that I have now is closer to the color I want. Should I still change to Bootstrap's colors?

Reply:
For consistency, bootstrap colors are better unless you have very strong stylistic feelings about it being a bright red... I think they will still get the message even if it is a darker red.

Reply:
The red wasn't the problem - the issue was that bootstrap's warning color didn't look like a warning color. It was too gray/dark imo.